config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -50,3 +50,46 @@ presubmits:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-kubernetes-e2e-capz-windows-containerd
       testgrid-num-columns-recent: '30'
+  kubernetes-sigs/windows-testing:
+  - name: pull-e2e-capz-containerd-windows-2022-extension
+    interval: 3h
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'azure/.*'
+    decoration_config:
+      timeout: 3h
+    path_alias: k8s.io/windows-testing
+    branches:
+      - master
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-windows-private-registry-cred: "true"
+      preset-capz-windows-common-main: "true"
```

Comment:
should this be `preset-windows-common-pull` instead of `preset-windows-common-main`?

Reply:
This isn't a presubmit against k/k.  We are using k/k CI builds so it's more accurate to be using the CI `preset-capz-windows-common-main` I think.

Reply:
got it